awx/ui_next/src/locales/zh/messages.po

```diff
@@ -8843,11 +8843,11 @@ msgstr "周"
 
 #: components/Schedule/shared/FrequencyDetailSubform.jsx:449
 msgid "Weekday"
-msgstr "周中日"
+msgstr "周内"
```

Comment:
@weidongkl for the translation of "Weekday", perhaps it might be better to translated as "工作日" or "周中"?  This string is describing the days during the work week (Monday - Friday).  This was the feedback from one of our internal translators.  What do you think?


 if it means "Monday to Friday"

Reply:
I agree that  "工作日" are the better  translation

Reply:
Ok, great.  Thank you @weidongkl!